utils/db.ts

```diff
@@ -100,6 +100,73 @@ export async function getItemCommentsCount(itemId: string) {
   return count;
 }
 
+interface InitVote {
+  userId: string;
+  itemId: string;
+  voteId?: string;
```

Comment:
Hm... I'm starting to think that the `voteId` property is not needed. If the entry exists for a given user or item, I think that's all that's needed. We could instead have the value of the entry be something arbitrary, simple but not misleading (not `null`). Perhaps, `undefined`? We'd also have to comment on the arbitrary nature of the value. WDYT?

Reply:
Ya, `userId` and `itemId` are enough to indicate that a vote has been cast.


Hmm.. I am not sure I understand this. Could you explain it in more detail or show me some examples?
>  We could instead have the value of the entry be something arbitrary, simple but not misleading (not null). Perhaps, undefined?

Reply:
The value would be arbitrary because it's never read. Whether the key exists is all we care about. Hence, the value can be `undefined`.

Reply:
I am currently removing `voteId` since it appears to have no applicable use cases ATM.